atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -54,6 +58,27 @@ static synchronized void setLogArbitrator(KeyValueServiceLogArbitrator arbitrato
     }
 
     /**
+     * Partition the given {@code tableReferences} in a pair of safe and unsafe tables for logging.
+     * The first element of the pair are the safe to log tables, while the second element of the pair are the unsafe.
+     */
+    public static Pair<Arg<List<TableReference>>, Arg<List<TableReference>>> tablesRef(
```

Comment:
Let's call this `tableRefs` instead of `tablesRef` - it's short for "table references".

Also, using `Pair` here is a bit lazy (we need to remember that the safe tables go on the `lhSide`).